plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryCliProvider.kt

```diff
@@ -0,0 +1,127 @@
+package io.sentry.android.gradle
+
+import org.gradle.api.Project
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.util.Properties
+
+internal object SentryCliProvider {
+    /**
+     * Return the correct sentry-cli executable path to use for the given project.  This
+     * will look for a sentry-cli executable in a local node_modules in case it was put
+     * there by sentry-react-native or others before falling back to the global installation.
+     */
+    @JvmStatic
+    fun getSentryCliPath(project: Project): String {
+        // If a path is provided explicitly use that first.
+        project.logger.info("[sentry] Searching cli from sentry.properties file...")
+
+        searchCliInPropertiesFile(project)?.let {
+            project.logger.info("[sentry] cli Found: $it")
+            return@getSentryCliPath it
+        } ?: project.logger.warn("[sentry] sentry-cli not found in sentry.properties file")
+
+        // in case there is a version from npm right around the corner use that one.
+        // This is the case for react-native-sentry for instance.
+        project.logger.info("[sentry] Searching cli inside ../node_modules/...")
```

Comment:
I'm not entirely sure and need to confirm, but I guess all the searching for react-native folders (node_modules) are not used anymore, the RN SDK has its own Gradle tasks now, we don't need to keep compatibility anymore.
Also, this plugin already ships sentry-cli so it's not needed to look for one unless there's a special need I'm not aware of.

https://github.com/getsentry/sentry-react-native/blob/master/sentry.gradle

@bruno-garcia @jennmueng can you confirm that?

Reply:
I've confirmed it, we could remove this part of searching for sentry-cli over node_modules, which means killing this use case, removing the method `searchCliInNodeModules`, etc...